catalog/app/components/Markdown/Markdown.js

```diff
@@ -63,6 +80,8 @@ const adjustLinks = (md) => {
   md.renderer.rules.link_open = (tokens, idx) => {
     const t = tokens[idx];
     const title = t.title ? ` title="${escape(t.title)}"` : '';
+    // TODO: sign
```

Comment:
what is this?

Reply:
i think we need to sign the links as well, not only images, so i've left this TODO

Reply:
correct